tests/lib/tools/snaps-state

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+show_help() {
+    echo "usage: make-snap [snap-name]"
```

Comment:
Perhaps just `snap-state pack` instead (to avoid repeating the word snap).

It would be good to explain in usage, that packed snaps are cached and reused. As otherwise this is no different from plain `snap ...`.

Reply:
I updated the names: pack-local install-local and install-local-as are the available commands.